src/components/Button/Button.css

```diff
@@ -0,0 +1,27 @@
+.button {
+    border-radius: 10px;
+    width: 40px;
+    height: 40px;
+    align-items: center;
+    justify-content: center;
+    display: flex;
+    border: 0;
+}
+
+.button--primary {
+    background-color: #FB76A1;
+}
+
+.button--secondary {
+    background-color: #21212B;
+    color: white
+}
+
+.button--special {
+    cursor: pointer;
```

Comment:
I would move the `cursor: pointer` property to `.button` selector so it applies to all the variants.